www/src/components/_doc-elements.scss

```diff
@@ -118,3 +118,183 @@
     }
   }
 }
+
+.pgn-doc__box-shadow--title {
+  font-size: 1.375rem;
+  line-height: 1.25;
+}
+
+.pgn-doc__box-shadow-cells {
+  display: grid;
+  grid-template-columns: repeat(5, 6.25rem);
+  grid-template-rows: repeat(5, 6.25rem);
+  grid-gap: 3.75rem 4.125rem;
+
+  .pgn-doc__box-shadow-cell {
+    width: 6.25rem;
+    height: 6.25rem;
+    cursor: copy !important;
+
+    &:focus,
+    &:hover,
+    &:focus-visible {
+      border: $border-width solid $dark;
+    }
+  }
+}
+
+[dir="rtl"] .pgn-doc__box-shadow-wrapper {
+  .pgn-doc__box-shadow-level-titles {
+    .pgn-doc__box-shadow-level-title {
+      text-align: right;
+    }
+  }
+
+  .pgn-doc__box-shadow-side-titles {
+    .pgn-doc__box-shadow-side-title {
+      text-align: right;
+
+      &:not(:last-child) {
+        margin-left: 4.125rem;
+        margin-right: 0;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+}
+
+.pgn-doc__box-shadow-wrapper {
+  overflow-x: scroll;
+
+  .pgn-doc__box-shadow-level-titles {
+    .pgn-doc__box-shadow-level-title {
+      font-size: $font-size-base;
+      line-height: $line-height-base;
+      width: 8.125rem;
+      height: 6.25rem;
+      text-align: left;
+      margin-bottom: 3.75rem;
+      white-space: nowrap;
+    }
+  }
+
+  .pgn-doc__box-shadow-side-titles {
+    display: flex;
+    min-width: 57rem;
+
+    .pgn-doc__box-shadow-side-title {
+      font-size: $font-size-base;
+      line-height: $line-height-base;
+      width: 6.25rem;
+      height: 3rem;
+
+      &:not(:last-child) {
+        margin-right: 4.125rem;
+      }
+
+      &:first-child {
+        width: 8.125rem;
+        margin-right: 0;
+      }
+    }
+  }
+}
+
+.pgn-doc__box-shadow-generator {
+  display: flex;
+  flex-direction: column;
+
+  .pgn-doc__box-shadow-generator--preview {
```

Comment:
Can you rename to BEM?
Example: `pgn-doc__box-shadow-generator__preview`, `pgn-doc__box-shadow-generator__preview-box`